test/kotlin/integration/StatFlushIntegrationTest.kt

```diff
@@ -49,9 +49,12 @@ class StatFlushIntegrationTest {
     val countDownLatch = CountDownLatch(1)
     engine = EngineBuilder()
       .addLogLevel(LogLevel.DEBUG)
-      .addStatsDPort(8125)
       // Really high flush interval so it won't trigger during test execution.
       .addStatsFlushSeconds(100)
+      .addExtraStatSinks(listOf("""{ name: envoy.stat_sinks.statsd,
```

Comment:
any chance that we could have a similar test for Swift?

Reply:
I can do one that just initializes one with the statsd sink without any extra tests just to see if the configuration bits work right

Reply:
that's better than nothing but is it possible to launch a mock server in the swift test (we have something like that in the Java/kotlin one) and see whether a metric is received by it? If that's easy to do then it should be considered to be outside of the scope of this PR

Reply:
Talked offline, not that easy to do since we need a statsd server written in Swift